src/cockpit/389-console/src/lib/database/localPwp.jsx

```diff
@@ -702,10 +702,10 @@ class CreatePolicy extends React.Component {
                                             isDisabled={!this.props.passwordchecksyntax}
                                         />
                                     </GridItem>
-                                    <GridItem className="ds-label" offset={5} span={3} title={_("The maximum number of times the same character can sequentially appear in a password (passwordMaxRepeats).")}>
+                                    <GridItem className="ds-label" offset={5} span={3}>
                                         {_("Max Sequence Sets")}
                                     </GridItem>
-                                    <GridItem span={1}>
+                                    <GridItem span={1} title={_("The maximum number of times the same character can sequentially appear in a password (passwordMaxRepeats).")}>
```

Comment:
All of these "title" updates need to also be reflected in "ja.po" file.  Otherwise the language translation will fail